packages/share_plus/share_plus/android/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
       </provider>
       <!-- This manifest declared broadcast receiver allows us to use an explicit
            Intent when creating a PendingItent to be informed of the user's choice -->
-      <receiver android:name=".SharePlusPendingIntent" android:exported="true">
+      <receiver android:name=".SharePlusPendingIntent" android:exported="false">
```

Comment:
Do we know why it was exported ? Was it on purpose or just a miss / was created exported by default?

Reply:
Is it that big deal? You can ping author and ask. I guess, it it because the author isn't a mobile dev, but I don't see why we should care about it, considering that the issue is resolved.